.gitlab-ci/terraform.yml

```diff
@@ -38,8 +38,7 @@
   script:
     - cd tests && make create-${CI_PLATFORM} -s ; cd -
     - ansible-playbook tests/cloud_playbooks/wait-for-ssh.yml
```

Comment:
I think you can remove these rows as well, as they are run in `tests/scripts/testcases_run.sh`?

Reply:
Good idea, I fixed it in the latest commit.